frontend/awx/resources/projects/ProjectPage/ProjectPage.tsx

```diff
@@ -50,6 +50,8 @@ export function ProjectPage() {
       { label: t('Access'), page: AwxRoute.ProjectAccess },
```

Comment:
Can we get rid of this "Access" tab and the accompanying route/navigation please? This is replaced by the User Access and Team Access tabs.

Reply:
Fixed.

Reply:
@ZitaNemeckova  Still seeing the redundant Access tab, is there a pending commit?